### ISSUE TITLE
How do I tell if a file is a directory from FileInfo

### ISSUE THREAD

**Issue by lpil** [bcpeinhardt/simplifile#40]
Hello! How do I do this?

I think it might be using the `mode` int, but how data is encoded into that int is not documented.

> File mode that indicates the file type and its permissions. For example, in Unix and Linux, a mode value of 33188 indicates a regular file and the permissions associated with it (read and write for the owner, and read-only for others, in this case).

Thanks,
Louis

**Comment by lpil** [comment]
That's great! Thank you.

I still think it would be useful to document what this field means, I don't know how to use it presently.

**Comment by bcpeinhardt** [comment]
I agree. I'm gonna add the relevant documentation, in the meantime just for you check out https://www.man7.org/linux/man-pages/man7/inode.7.html (scroll down to "The file type and mode")

**Comment by bcpeinhardt** [comment]
Completely let this slip. For the moment I've just linked to that man page in the docs. I need to time block check ins with my existing repos.

**Comment by Daniel-Shunom** [comment]
Hi everyone! Is this still being worked on?

**Comment by bcpeinhardt** [comment]
Hi @Daniel-Shunom ! 
If you'd like to add this documentation the PR would be more than welcome 😎 If you just need the info the link pasted above should have what you need, but feel free to ask a specific question!
